src/components/Drawer/types.ts

```diff
@@ -0,0 +1,39 @@
+import { ReactNode } from 'react';
+
+
+export type Anchors = 'top' | 'right' | 'bottom' | 'left';
+
+
+export interface Props {
+
+    /**
+     * Where to position the drawer
+     */
+    anchorFrom: Anchors,
+
+    /**
+     * Children passed to the component (JSX, String)
+     */
+    children?: ReactNode,
+
+    /**
+     * Custom class to provide component
+     */
+    className?: string,
+
+    /**
+     * Controlled prop to determine drawer open state
+     */
+    isOpen?: boolean,
+
+    /**
+     * Default open state drawer
+     */
+    isDefaultOpen?: boolean,
```

Comment:
This is no longer needed since there is no uncontrolled state, right?

Reply:
fixed